app/views/Posts/index.html.erb

```diff
@@ -1,16 +1,27 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>BlogApp</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+<div class="user-card">
+  <div></div>
+  <div>
+    <h2><%= @single_user.name %></h2> 
+    <span>Number Of Posts: <%= @single_user.post_counter %></span>
+  </div>
+</div>
 
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_importmap_tags %>
-  </head>
-
-  <body class="body">
-  <h1>Here is a list of posts for a given user</h1>
-  </body>
-</html>
\ No newline at end of file
+<div>
+  <% @user_posts.each do |post| %>
+    <div>
+      <div>
+        <h3><%= post.title %></h3>
+        <p><%= post.text %></p>
+        <span>
+          Comments: <%= post.comments_counter %>
+          Likes: <%= post.likes_counter %>
+        </span>
+      </div>
+      <div class="post-comment">
+        <% post.recent_comments.each do |comment| %>
+          <h3><%= comment.author.name %>: <%= comment.text%></h3>
+        <% end %>
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly ensure that the `app/views/Posts/index.html.erb` view page is working as it should display this **[wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png)**. Right now it shows the below error, so impossible to check the result:
![Screenshot from 2023-01-27 04-24-15](https://user-images.githubusercontent.com/83042742/214997327-ff366ea8-1593-47d7-8bcb-d36c14131e74.png)

Reply:
Needs attention!